runtime/kusama/src/lib.rs

```diff
@@ -79,7 +79,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	impl_name: create_runtime_str!("parity-kusama"),
 	authoring_version: 2,
 	spec_version: 1034,
-	impl_version: 0,
+	impl_version: 1,
```

Comment:
Are we sure that nothing changed in runtime by pulling latest substrate?

Reply:
yes - and in any case 1034 never made it on to the network.